data/tutorials/platform/1_04_hierarchy_builder_docs.md

```diff
@@ -0,0 +1,18 @@
+---
+id: "hierarchy-builder-docs"
+title: "Documentation of Hierarchy Builder"
+description: |
+  Documentation of Hierarchy Builder
+category: "Documentation"
+---
+
+## What is Hierarchy Builder?
+
+Hierarchy Builder is a plugin for Rocq that provides high level commands to
+declare a hierarchy of structures. It turns a synthetic description of a
+structure into the low-level Rocq commands required to declare the structure,
+while ensuring that a few good properties hold.
+
+## Documentation of Hierarchy Builder
+
+-   Hierarchy Builder [source code](https://coq.inria.fr/platform-docs/Tutorial_hierarchy_builder.v)
```

Comment:
```suggestion
-   Hierarchy Builder [source code](https://rocq-prover.org/platform-docs/Tutorial_hierarchy_builder.v)
```

Why no link to an interactive version? It's broken because it doesn't have the HB dependency? We could try to fix that instead.